bcipy/helpers/task.py

```diff
@@ -370,186 +373,153 @@ def get_key_press(
-                   eeg_data: np.array,
-                   fs: int, k: int, mode: str,
+                   eeg_data: np.ndarray,
+                   fs: int,
+                   trials_per_inquiry: Optional[int] = None,
                    offset: float = 0,
-                   channel_map: tuple = DEFAULT_CHANNEL_MAP,
-                   trial_length: float = 0.5) -> tuple:
-    """Trial Reshaper.
-
-    Trail reshaper is used to reshape trials, based on trial target info (target, non-target),
-        timing information (sec), eeg_data, sampling rate (fs), down-sampling rate (k),
-        mode of operation (ex. 'calibration'),
-        offset (any calculated or hypothesized offsets in timings),
-        channel map (which channels to include in reshaping) and
-        trial length (length of reshaped trials in secs)
-
-    In all modes > calibration, we assume the timing info is given in samples not seconds.
-
-    :return (reshaped_trials, labels, num_of_inquiries, trials_per_inq)
-
-        reshaped_trials is a 3 dimensional np array where the first dimension
-            is the channel, second dimension the trial/letter, and third
-            dimension is time samples. So at 300hz and a presentation time of
-            0.2 seconds per letter, the third dimension would have about 60
-            samples.
-            If this were a json structure it might look like:
-                {
-                    'ch1': {
-                        'B' : [-647.123, -93.124, ...],
-                        'T' : [...],
-                        ...
-                    },
-                    'ch2': {
-                        'B' : [39.60, -224.124, ...],
-                        'T' : [...],
-                        ...
-                    },
-                    ...
-                }
+                   channel_map: List[int] = DEFAULT_CHANNEL_MAP,
+                   trial_length: float = 0.5) -> Tuple[np.ndarray, np.ndarray]:
+    """Extract trial data and labels.
+
+    Args:
+        trial_target_info (list): labels each trial as "target", "non-target", "first_pres_target", etc
+        timing_info (list): Timestamp of each event in seconds
+        eeg_data (np.ndarray): shape (channels, samples) preprocessed EEG data
+        fs (int): sample rate of preprocessed EEG data
+        trials_per_inquiry (int, optional): unused, kept here for consistent interface with `inquiry_reshaper`
+        offset (float, optional): Any calculated or hypothesized offsets in timings. Defaults to 0.
+        channel_map (tuple, optional): Describes which channels to include or discard. Defaults to DEFAULT_CHANNEL_MAP.
+        trial_length (float, optional): [description]. Defaults to 0.5.
+
+    Returns:
+        trial_data (np.ndarray): shape (channels, trials, samples) reshaped data
+        labels (np.ndarray): integer label for each trial
+
+    TODO ??? In all modes > calibration, we assume the timing info is given in samples not seconds.
```

Comment:
That assumption is not correct. We only use this on load in all data is stored in the same fashion.

Reply:
Thanks for catching this. This statement was in the previous doc and I couldn't understand what it meant so I left a TODO. I'll delete it.